docs/Index.tsx

```diff
@@ -121,6 +125,7 @@ class App extends React.Component<any, any> {
             <Provider store={ReactVaporStore}>
                 <div className='coveo-form'>
                     {[
+                        {component: CollapsibleInfoBoxExamples, componentName: 'CollapsibleInfoBox'},

```

Comment:
could you remove all the `Examples` suffix in `componentName` it will mirror better the name of our components

Reply:
thought so too